ts/torch_handler/base_handler.py

```diff
@@ -59,6 +59,24 @@ def check_pt2_enabled():
         )
 
 
+def check_torch_xla_enabled() -> bool:
```

Comment:
@lxning another good candidate for your new config change, it might be possible that a user has xla installed but doesnt want to necessarily comile the model with XLA

Reply:
@msaroufim yes. the model yaml config can make this much easier